Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/competition/RedRightPreload.java

```diff
@@ -0,0 +1,83 @@
+package org.firstinspires.ftc.team6220_PowerPlay.competition;
+
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+
+import org.firstinspires.ftc.team6220_PowerPlay.BaseAutonomous;
+import org.firstinspires.ftc.team6220_PowerPlay.Constants;
+
+@Autonomous(name = "Red Right Preload", group = "Competition")
+public class RedRightPreload extends BaseAutonomous {
+
+    @Override
+    public void runOpMode() throws InterruptedException {
+        initialize();
+        waitForStart();
+
+        // grab cone
+        servoGrabber.setPosition(Constants.GRABBER_CLOSE_POSITION);
+        sleep(1000);
+
+        // drive slides upwards a few inches
+        driveSlidesAutonomous(Constants.SLIDE_LOW);
+        sleep(500);
+
+        int signal = detectSignal();
+
+        // drive to furthest tile in our quadrant
+        driveInches(0, 54);
+        sleep(500);
+
+        // turn to 45 degrees
+        turnToAngle(45);
+        sleep(500);
+
+        // drive slides to high junction
+        driveSlidesAutonomous(Constants.SLIDE_HIGH);
+        sleep(500);
+
+        // drive forwards to high junction
+        driveInches(0, 5 * Math.sqrt(2));
+        sleep(500);
+
+        // drive slides to high junction
```

Comment:
Same suggestion

Reply:
This comment is no longer relevant as the code has been deleted